### PR TITLE
Use flycast for pg attach if it exists

### PR DESCRIPTION
**Pull request by DAlperin**

### CODE REVIEW

File: internal/command/postgres/attach.go [hunk 1]
```diff
@@ -101,9 +101,23 @@ func runAttach(ctx context.Context) error {
...
 
+	pgAppFull, err := client.GetApp(ctx, pgAppName)
+	if err != nil {
+		return fmt.Errorf("failed retrieving postgres app %s: %w", pgAppName, err)
+	}
+
+	var flycast *string
+
+	for _, ip := range pgAppFull.IPAddresses.Nodes {
+		fmt.Println(ip)
```
Comment (davissp14) [inline internal/command/postgres/attach.go:112]: We can ditch the println here.

File: internal/command/postgres/attach.go [hunk 2]
```diff
@@ -101,9 +101,23 @@ func runAttach(ctx context.Context) error {
...
+		if ip.Type == "private_v6" {
+			flycast = &ip.Address
```
Comment (davissp14) [inline internal/command/postgres/attach.go:114]: Can there only be one?

**Review by davissp14 — APPROVED** [review]
Few questions, but if nothing critical. LGMT

File: internal/command/postgres/attach.go (hunk 1 above)
Comment (DAlperin) [reply internal/command/postgres/attach.go:112]: oh oops. from debugging the web issue

**Pull request merged**